charaData.json

```diff
@@ -659,6 +659,50 @@
         "imageURL": "./charaimgs/3040169000_01.png",
         "en": "Nier"
     },
+    "タヴィーナ": {
+        "name": "タヴィーナ",
+        "ja": "タヴィーナ",
+        "element": "fire",
+        "type": "attack",
+        "race": "human",
+        "sex": "female",
+        "fav1": "bow",
+        "fav2": "axe",
+        "support": "none",
+        "support2": "none",
+        "support3": "none",
+        "minhp": "245",
+        "hp": "1300",
+        "minattack": "1780",
+        "attack": "9550",
+        "baseDA": 7.0,
+        "baseTA": 3.0,
+        "ougiRatio": 4.5,
+        "imageURL": "./charaimgs/3040234000_01.png",
+        "en": "Tabina"
+    },
+    "勇者と姫君 スタン＆アリーザ": {
+        "name": "勇者と姫君 スタン＆アリーザ",
+        "ja": "勇者と姫君 スタン＆アリーザ",
+        "element": "wind",
+        "type": "attack",
+        "race": "erune",
```

Comment:
"race": "erune/draf",

Reply:
global_const.js had entry for multiple races "human/erune" raceTypes

Reply:
we need "male/female" entry for sexTypes as well ?

Reply:
I've implemented chara filtering #361 that will use multiple value in the list.

Reply:
Yeah when coping the html data and converting it into csv, jp wiki probably had erune only in their list instead of both. Should start proof reading their data again before copying.